ansible/tasks/stage2-setup-postgres.yml

```diff
@@ -30,6 +34,14 @@
       when: is_psql_oriole or is_psql_17 and stage2_nix
       become: yes
 
+    - name: Remove pgjwt from supautils.conf if 15 build
+      ansible.builtin.command:
+        cmd: >
+          sed -i 's/ pgjwt,//g' 
+          /etc/postgresql-custom/supautils.conf
+      when: is_psql_15 and stage2_nix
+      become: yes
```

Comment:
Q: Can this be tried locally?

If not, we're planning to remove ansible later right?

Reply:
@steve-chavez yes we are planning to remove ansible later. This can be tried locally, and will be part of our local testing env I'll add a change for that. 

I am starting with the auth project to migrate package and config steps out of ansible. Then likely postgres will be next, fwiw.